packages/volto/src/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -42,7 +42,8 @@ const blockPropsAreChanged = (prevProps, nextProps) => {
   return isEqual(prev, next);
 };
 
-const applyDefaults = (data, root) => {
+const applyDefaults = (data, root, block_query) => {
```

Comment:
Javascript uses camelCase. So it's better to use `blockQuery`. For other variables as well.